skymp5-client/src/modelSource/remoteServer.ts

```diff
@@ -226,6 +248,25 @@ export class RemoteServer implements MsgHandler, ModelSource, SendTarget {
   }
 
   createActor(msg: messages.CreateActorMessage): void {
+    if (skipFormViewCreation(msg)) {
+      const refrId = msg.refrId;
+      onceLoad(refrId, (refr: ObjectReference) => {
```

Comment:
`refrId` can't be undefined here but `msg.refrId` can. We need guard or default value.

Reply:
not sure I understand

Reply:
`onceLoad` can be used only with a `number` type. `messages.CreateActorMessage.refrId` is an optional field, so it can be undefined.